running/safari.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Use Cockpit with Safari on an iPhone or iPad
+---
+
+Safari on iOS and iPadOS 14.5 has experimental support for secure WebSocket sessions, which Cockpit needs to function.
+
+Unfortunately, proper WebSocket support is currently turned off by in 14.5. Until Apple makes this the default, here are instructions for a workaround to make iPhones work with Cockpit.
```

Comment:
My understanding that this is for self-signed/invalid certificates only? If so you should probably lay out *all* the solutions, since something like Let's Encrypt is still probably a better idea. Something like:

```suggestion
Safari's default WebSocket framework on iOS, iPadOS, and ARM-based Macs does not support the WebSocket Secure sessions which Cockpit needs to function when using a self-signed or otherwise invalid certificate, even when they are manually marked as trusted. [Installing a valid certificate]({{ site.baseurl }}/guide/latest/guide.html) should resolve this issue.

As an alternative workaround, you can enable Apple's future/experimental WebSocket framework which does work as expected:
```

Reply:
Oh, right! Thanks for the catch!

I was so eager to publish the workaround, I forgot to mention that this is for self-signed certificates.

Thanks for the wording; I've applied the suggestion.